album/SECL-2537.toml

```
@@ -0,0 +1,24 @@
+[album]
+album_id = "64600c4a-8b4f-4719-bd30-3fa77cec552e"
+title = "Prover / Tell me"
+artist = "milet"
+date = 2020-02-17
+type = "normal"
+catalog = "SECL-2537"
+tags = ["TVアニメ「Fate/Grand Order -絶対魔獣戦線バビロニア-」"]
+
+[[discs]]
+catalog = "SECL-2537"
+
+[[discs.tracks]]
+title = "Prover"
+tags = ["ED"]
+
+[[discs.tracks]]
+title = "Tell me"
```

```suggestion
title = "Tell me"
tags = ["TVアニメ「Fate/Grand Order -絶対魔獣戦線バビロニア-」"]
```

M2「Tell me」・・・TVアニメ『Fate/Grand Order -絶対魔獣戦線バビロニア-』スペシャルテーマソング